src/components/text-input/index.md.njk

```diff
@@ -59,6 +59,19 @@ Fluid width inputs will resize with the viewport.
 
 {{ example({group: "components", item: "text-input", example: "input-fluid-width", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
+### Numbers
+
+If it’s likely that the user will need to enter a number and you want to bring up the numeric keypad on a mobile device, use the following approach:
```

Comment:
@StephenGill Following the example of the [autocomplete section](https://design-system.service.gov.uk/components/text-input/#use-the-autocomplete-attribute), I wonder if this should be something like:
> To enable the numeric keypad, set the `input` attribute to `numeric` and the `pattern` attribute to `[0-9]*`. See how to do this in the HTML and Nunjucks tabs in the following example.

Since we display the code snippet for `input type=”number”` further down it probably makes sense to call out the attributes here too for doing it correctly.